app/src/main/res/layout/sheet_phone_mask.xml

```diff
@@ -28,7 +28,7 @@
     android:gravity="center"
     android:maxLines="1"
     android:textAppearance="@style/Clinic.V2.TextAppearance.Body2Left.Numeric.Grey1"
-    tools:text="+91 9876543210" />
```

Comment:
Curious, why are we removing this?

Reply:
Because this is not in sync with the actual implementation. In the production code, we do not prepend the country dialling code to the number. I'm guessing this was changed after the initial feature was built, but we did not change the layout placeholder.